lib/winapi/.clang-format

```diff
@@ -0,0 +1,21 @@
+BasedOnStyle: Microsoft
+ColumnLimit: 0
+SortIncludes: false
+IndentCaseLabels: true
+AlignTrailingComments: false
+
+SpaceBeforeParens: Custom
+SpaceBeforeParensOptions:
+  AfterControlStatements: true
+  AfterFunctionDefinitionName: true
+  AfterFunctionDeclarationName: true
+
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterControlStatement: false
+  AfterExternBlock: false
+  BeforeElse: false
+  AfterUnion: false
```

Comment:
Looks a bit weird imho to have `struct` and `union` formatted differently, consistency would be nicer. I used to prefer having opening braces on their own line, but I don't have strong feelings either way, so I'm fine with whatever style you choose.

Reply:
> AfterUnion: false

Ill make this true (i.e remove the line). I think this was just a case of a mixed bag